### PR TITLE
Created login

### PR DESCRIPTION
**Pull request by AKILBEK-UR**

### CODE REVIEW

File: src/modules/auth/auth.module.ts [hunk 1]
```diff
@@ -3,8 +3,6 @@ import { AuthService } from './auth.service';
 import { AuthController } from './auth.controller';
 import { JwtModule } from '@nestjs/jwt';
 import { UsersModule } from '../users/users.module';
-import { UsersRepository } from '../users/users.repository';
-import { DataSource } from 'typeorm';
 
 @Module({
   imports: [
```
Comment (purple-mountain) [inline src/modules/auth/auth.module.ts:8]: I think we do not have AUTH_TOKEN_SECRET or AUTH_TOKEN_EXPIRES_IN env variables. You should change those names to AUTH_ACCESS_TOKEN_SECRET or AUTH_ACCESS_TOKEN_EXPIRES_IN

File: src/modules/auth/auth.service.ts [hunk 2]
```diff
@@ -35,6 +36,27 @@ export class AuthService {
...
+      throw new BadRequestException('Invalid email or password');
+    }
+    const payload = { sub: user.id, userEmail: user.email };
+    const accessToken = await this.generateAccessToken(payload);
+
+    return { accessToken };
+  }
   async generateAccessToken(payload: { sub: string; userEmail: string }) {
     const accessToken = await this.jwtService.signAsync(payload, {
       secret: this.configService.get('AUTH_ACCESS_TOKEN_SECRET'),
```
Comment (purple-mountain) [inline src/modules/auth/auth.service.ts:62]: After providing expiresIn and secret options inside auth.module.ts during import, you can omit options and just use 
```js
    const accessToken = await this.jwtService.signAsync(payload);
```
when you are creating access tokens.

**Review by purple-mountain — CHANGES_REQUESTED** [review]
Later, we will also need to add refresh tokens and cookies, but for now I think it is good enough.

**Review by mukhammadjon-ss — APPROVED** [review]
After resolving comments you can merge it.

**Review by msirojiddinov — APPROVED** [review]

**Review by purple-mountain — APPROVED** [review]

**Pull request merged**